### DISCUSSION TITLE
CSS to external file?

### DISCUSSION THREAD

**Discussion by riekusr** [gildas-lormeau/SingleFile#906] [Ideas]
Is there any way we can save a file with no inline styles and 'auto create' classes and put the css in a external file?

anyone know any other script I can use for this use case? Google only talks about external css to inline conversion.

**Comment by ghost** [comment]
Hi riekusr! This is possible I believe, for example there is CSS Used which is an extension that extracts css from html. I believe that if SingleFile has CSS Used support, that would be possible for what you are requesting. source-code: https://github.com/painty/CSS-Used-ChromeExt
